src/databricks/labs/ucx/cli.py

```diff
@@ -504,5 +505,43 @@ def create_catalogs_schemas(w: WorkspaceClient, prompts: Prompts):
     catalog_schema.create_catalog_schema()
 
 
+@ucx.command
+def cluster_remap(w: WorkspaceClient, prompts: Prompts):
+    """Re-mapping the cluster to UC"""
+    logger.info("Remapping the Clusters to UC")
+    installation = Installation.current(w, 'ucx')
+    cluster = ClusterAccess(installation, w, prompts)
+    cluster_list = cluster.list_cluster()
+    print(f"{'Cluster Name':<80}\t{'Cluster Id':<80}")
```

Comment:
assuming that the average screen width is 80 characters, isn't it too long?...

Reply:
<img width="1079" alt="image" src="https://github.com/databrickslabs/ucx/assets/127273819/fda6f02a-8be5-4788-a6b5-4a9c04366834">
Reduced the width to 50. It looks fine.